### PR TITLE
Made `positive-infinity.plus` and `negative-infinity.plus` work with multiple arguments

### PR DESCRIPTION
**Pull request by maxonfjvipon**
Closes: #1806

### CODE REVIEW

**Comment by maxonfjvipon** [comment]
@Graur please have a look

File: eo-runtime/src/main/eo/org/eolang/negative-infinity.eo [hunk 1]
```diff
@@ -59,48 +59,78 @@
...
-  [x] > minus
-    ^.plus (x.neg) > @
-
-    [x] > not-int
-      x.as-int > as-integer!
-      and. > @
-        as-integer.as-float.lt x
-        (as-integer.plus 1).as-float.gt x
+  [x...] > minus
+    ^.plus ((x.at 0).neg) > @
```
Comment (Graur) [inline eo-runtime/src/main/eo/org/eolang/negative-infinity.eo:124]: @maxonfjvipon Can you explain why we pass to `plus` object only the first element of the `x...` vararg array?

File: eo-runtime/src/main/eo/org/eolang/positive-infinity.eo [hunk 2]
```diff
@@ -61,43 +58,88 @@
...
     negative-infinity > @
 
+  # @todo #1806:30min. Make positive-infinity.minus and negative-infinity.minus
+  #  worked with multiple arguments without using objects from eo-collections.
+  #  See positive-infinity.plus and negative-infinity.plus as an example
   # Difference between $ and x
-  [x] > minus
-    ^.plus (x.neg) > @
+  [x...] > minus
+    ^.plus ((x.at 0).neg) > @
```
Comment (Graur) [inline eo-runtime/src/main/eo/org/eolang/positive-infinity.eo:130]: @maxonfjvipon Same here, I don't get it

**Review by Graur — CHANGES_REQUESTED** [review]

File: eo-runtime/src/main/eo/org/eolang/negative-infinity.eo (hunk 1 above)
Comment (maxonfjvipon) [reply eo-runtime/src/main/eo/org/eolang/negative-infinity.eo:124]: @Graur I added 3 pdd puzzles in `positive-infinity.eo` to implement these objects in the future: minus, times, div. `x...` in these objects are kind of placeholders now

File: eo-runtime/src/main/eo/org/eolang/positive-infinity.eo (hunk 2 above)
Comment (maxonfjvipon) [reply eo-runtime/src/main/eo/org/eolang/positive-infinity.eo:130]: @Graur see above

File: eo-runtime/src/main/eo/org/eolang/negative-infinity.eo (hunk 1 above)
Comment (Graur) [reply eo-runtime/src/main/eo/org/eolang/negative-infinity.eo:124]: @maxonfjvipon Yeah, I missed that, thanks!

**Review by Graur — APPROVED** [review]
@maxonfjvipon Looks like we need to close those puzzles before `eo` release
@yegor256 Please, take a look

**Comment by maxonfjvipon** [comment]
@Graur sure, I'll try to make them faster

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**